router/internal/jitterticker/jitterticker_test.go

```diff
@@ -0,0 +1,55 @@
+package jitterticker
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
You never tested `Stop()` and therefore never run into the panic 😄

Reply:
This is fixed and tested now.